container-toolkit/release-notes.md

```diff
@@ -6,9 +6,36 @@
 
 # Release Notes
 
-This document describes the new features, improvements, fixed and known issues for the NVIDIA Container Toolkit.
+This document describes the new features, improvements, fixes and known issues for the NVIDIA Container Toolkit.
 
-______________________________________________________________________
+## NVIDIA Container Toolkit 1.17.8
+
+This release of the NVIDIA Container Toolkit `v1.17.8` is a bugfix release.
+
+### Fixes and Features
+
+- Updated the ordering of Mounts in CDI to have a deterministic output. This makes testing more consistent.
+- Added NVIDIA_CTK_DEBUG envvar to hooks.
```

Comment:
```suggestion
- Added NVIDIA_CTK_DEBUG environment variable to hooks.
```

i think it makes sense to spell this out for clarity (unless envvar is something else)

Reply:
+1

Reply:
Sure. Let's use the full text instead.